src/commands/aksOpenKubectlPanel/aksOpenKubectlPanel.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from "vscode";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { KubectlDataProvider, KubectlPanel } from "../../panels/KubectlPanel";
+import { CommandResponse } from "../../plugins/shared/pluginResponses";
+import { getExtension } from "../utils/host";
+import { getKubectlCustomCommands } from "../utils/config";
+import { failed } from "../utils/errorable";
+import { createTempFile } from "../utils/tempfile";
+import { getReadySessionProvider } from "../../auth/azureAuth";
+import { selectClusterOptions, SelectClusterOptions } from "../../plugins/shared/clusterOptions/selectClusterOptions";
+import { checkExtension, handleExtensionDoesNotExist } from "../utils/ghCopilotHandlers";
+
+const GITHUBCOPILOT_FOR_AZURE_VSCODE_ID = "ms-azuretools.vscode-azure-github-copilot"
+
+export async function aksOpenKubectlPanel(_context: IActionContext, target: unknown) {
+
+    const checkGitHubCopilotExtension = checkExtension(GITHUBCOPILOT_FOR_AZURE_VSCODE_ID);
+
+    if(!checkGitHubCopilotExtension) {
+        handleExtensionDoesNotExist(GITHUBCOPILOT_FOR_AZURE_VSCODE_ID);
+        return;
+    }
+
+    const responseCode = (target as CommandResponse).code;
+    const sessionProvider = await getReadySessionProvider();
+
+    if(failed(sessionProvider)) {
+        vscode.window.showErrorMessage(sessionProvider.error);
+        return;
+    }
+
+    const cluster = await selectClusterOptions(sessionProvider.result, [SelectClusterOptions.NewCluster]);
+
+    if (failed(cluster)) {
+        vscode.window.showErrorMessage(cluster.error);
+        return;
+    }
+
+    if (typeof cluster.result === "boolean") {
```

Comment:
💡 Early suggestion: (Please note, I can add this as separate PR for you as collab if you are busy)

```*.ts
if (Boolean(cluster.result) === false) {
    vscode.window.showErrorMessage("No cluster selected. Please select a valid cluster.");
    return;
}
```

Reply:
I tried adding this piece of code, but got errors involving: 48:9 error This expression unnecessarily compares a boolean value to a boolean instead of using it directly. Decided to use it directly